README.md

```diff
@@ -111,6 +111,12 @@ curl --include \
 4. localhost:8006/paid_invoice_from_xendit (callback url) get the paid invoice data.
 ```
 
+# Issue credit card refund
+```
+cd src/
+php examples/issue_credit_card_refund.php [amount] [external_id] [optional-idempotency-key]
```

Comment:
Also, I think we need to start adding example because some people often confused even if we have write optional idempotency key.

Example 1 (without idempotency key)
Example 2 (with idempotency key)

Benefits:
1. Clear docs
2. Easier for us to do testing, which we must at least testing those 2 cases.

Thoughts? Would like to implement that in this commit or later? @afik @jcontini